linkml/generators/pythongen.py

```diff
@@ -769,6 +774,38 @@ def is_key_value_class(self, range_name: DefinitionName) -> bool:
                 return len(rng.slots) - len(pkeys) == 1
         return False
 
+    def gen_construct(self, cls: ClassDefinition) -> Optional[str]:
```

Comment:
It's a small thing, but I got confused on get_construct thinking that a `construct` was a linkml concept I wasn't familiar with yet, would it be alright to rename this to `get_constructor`?

Reply:
ok, done and rebased